recipes/holoviews-1.5.0/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: holoviews
+  version: 1.5.0
+
+source:
+  path: .
```

Comment:
You need to add a `fn`/`url`/`sha256` field like [this](https://github.com/conda-forge/pint-feedstock/blob/master/recipe/meta.yaml#L7-L10).